public/css/login-registration.css

```diff
@@ -3,11 +3,23 @@
 	height: 100%;
 }
 
+/* UPPERCASE TEXT */
+.register,
+.form-control,
+.login,
+.ideahacks-form label,
+a,
+h1 {
+	text-transform: uppercase;
+}
+
```

Comment:
This rule currently applies to the input boxes as well, which makes users think their caps lock is on. I think just removing `.form-control` would fix this, but double-check that it doesn't break anything else.

Reply:
This makes the placeholder text capital case, is that fine or should that be uppercase still? Here's a picture for reference. 
Capital case:
![image](https://user-images.githubusercontent.com/57082175/129453647-20b3cf84-67a8-4e23-9267-95281e19364d.png)

Uppercase:
![image](https://user-images.githubusercontent.com/57082175/129453656-d2c8909f-b666-4baa-8f6e-5312c32ab4b2.png)
Personally I think capital case looks better here so I've kept it that way for my commit.

Reply:
I agree that capital case is better here; there's already a lot of bold/thick/uppercase text, and the capital case is closer to how the user will be typing, making them feel more comfortable.